src/components/Highlights/index.tsx

```diff
@@ -53,7 +53,22 @@ export function Cards({ from: cards }: { from: CardProps[] }) {
 		<section className="margin-top--lg margin-bottom--xl">
 			<div className="container">
 				<div className={clsx("margin-bottom--md")}>
-					<Heading as="h2">Our favorites ✨</Heading>
+					{ cards.length ? <Heading as="h2">Our favorites ✨</Heading> : <></> }
+
+					{ /* TODO: remove this once the plugin system is ready */ }
+					{ !cards.length && <>
+						<br/><br/>
+						<Heading as="h3" style={{ color: "blue" }}>
+							The plugin system is coming soon, stay tuned!
+						</Heading>
+
+						<br/>
+						<Heading as="h3" style={{ color: "red" }}>
```

Comment:
This doesn't look too great... 😆
![Screenshot 2023-11-06 at 16 24 51 (Arc)](https://github.com/yazi-rs/yazi-rs.github.io/assets/47499684/e1c89047-3cfe-498e-98d0-ca4d5cf3fb51)